<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="input_scan_edge_topic" default="/scan_edge" description="Edge points topic" />
  <arg name="input_scan_surface_topic" default="/scan_surface" description="Surface points topic" />
  <arg name="input_pose_topic" default="/pose" description="Lidar pose" />

  <arg name="node_name" default="lidar_feature_mapping" />

  <node pkg="lidar_feature_mapping" exec="lidar_feature_mapping" name="$(var node_name)" output="log">
    <remap from="scan_edge" to="$(var input_scan_edge_topic)" />
    <remap from="scan_surface" to="$(var input_scan_surface_topic)" />
    <remap from="pose" to="$(var input_pose_topic)" />
  </node>
</launch>
